src/pytds/tls.py

```diff
@@ -105,11 +105,19 @@ def validate_host(cert, name):
         ext = cert.get_extension(i)
         if ext.get_short_name() == b'subjectAltName':
             s = str(ext)
```

Comment:
Please refactor this code: extract logic inside this if branch into its own method `is_san_matching(san: str, host_name: str): bool`.

Also add unit tests for that method for following conditions:
* exact SAN match
* wildcard SAN match
* SAN mismatch

Reply:
I added a number of tests in tls_san_test.py, I hope thats correct this way?